tests/test_linter.py

```diff
@@ -283,6 +283,18 @@ def test_python_var_missing_right_curly_brace_two_vars(self):
         assert msgs[0].code == "E102"
         assert msgs[0].msg == 'missing right curly-brace: {0]" excede el tamano de {'
 
+        linted_entry = build_linted_entry(
+            "#: kitsune/questions/templates/questions/question_details.html:14\n"
+            'msgid "{q} | {product} Support Forum"\n'
+            'msgstr "{q} | {product}} foo bar"\n'
+        )
+
+        msgs = self.lintrule.lint(self.vartok, linted_entry)
+        assert len(msgs) == 1
+        assert msgs[0].kind == "err"
+        assert msgs[0].code == "E102"
+        assert msgs[0].msg == "missing right curly-brace: {product}} foo bar"
```

Comment:
Ugh? This error is wrong here: it's not missing a right curly-brace, it's missing a left...

I suppose this is a bug with the lint rule - it shouldn't care about this. If you really want to add a missing testcase, it should probably be something like `"{q} | {{product} foo bar`... but I'm not sure it'd pass.

Reply:
The error isn't wrong - it's missing a matching closing brace for the token (right) for the opening brace for the token (left).  `{{` and `}}` are just literal braces (according to the python string format spec) so not connected.  

(If it wasn't wrong then it implies a `{{` is required for a `}}`, but `{{` and `}}` don't need to be in pairs).

Reply:
Just as if the way to write a literal `{` was `\{` then the error for "\{product} foo" would be that the left curly brace was missing, not that a right `\}` was.